### ISSUE TITLE
Add priority to task during task creation

### ISSUE THREAD

**Issue by knappg** [klauscfhq/taskbook#68]
I find myself frequently creating tasks that I know I'd like to give a priority of 2 or 3. However, the way this is done now is two steps:

```
tb -t 'My created task'
tb -p @<task_id> 2
```

It'd be great if we could have an option on the `--task` option to do one of the following:
```
tb -t -p=2 'My created task'
tb -t -p 2 'My created task'
tb -t 'My created task' -p 2 
```

**Comment by 0x4eadac43** [comment]
You can do it with this syntax (example from documentation):
`$ tb -t @coding "Fix issue #42" p:3`
Note `p:3` at the end.

**Comment by klauscfhq** [comment]
More on how to define a priority level on task initialization; [`flight-manual#set-priority`](https://github.com/klauscfhq/taskbook#set-priority).
